profiles/api/oauth2.py

```diff
@@ -137,6 +141,12 @@ def _token() -> Response:
 
         _update_profile(profile, orcid_token)
 
+        # manually commit session here?
+        db.session.commit()
```

Comment:
perhaps this is good enough for now, it's a pain point to deal with sooner or later